Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractUse.cs

```diff
@@ -201,6 +203,10 @@ private void DoStartUseObject(object sender, ControllerInteractionEventArgs e)
 
             if (touchedObject != null && interactTouch.IsObjectInteractable(touchedObject))
             {
+                var vrobj = touchedObject.GetComponent<VRTK_InteractableObject>();
```

Comment:
i'd recommend naming this something else, even if it's just `vrObj`

looks a bit weird just all lowercase